docs/topics/external-packages.txt

```diff
@@ -0,0 +1,57 @@
+=================
+External packages
+=================
+
+Django ships with a variety of extra, optional tools that solve common
+problems. For easier maintenance and to trim the size of the codebase,
+during the course of Django development a few of those applications
+have been moved out to separate projects. A few examples follow.
+
+Localflavor
+~~~~~~~~~~~
+
+The ``localflavor`` app, a collection of utilities for particular
+countries or cultures, was removed from Django in 1.6. The new package
+is named ``django-localflavor`` and the official documentation can be
```

Comment:
Let's link the package names to the GitHub repos and the word "documentation" to the docs (probably don't need "official").